### PR TITLE
Add Digital.ai Deploy plugin to the microsite

### PR DESCRIPTION
**Pull request by sishwarya**
## Hey, I just made a Pull Request!

This PR updates the Plugin microsite to list the new dai-deploy Backstage plugin.

An itemized list of changes

Added deploy.yaml to /microsite/data/plugins/ and added a corresponding icon to /microsite/static/img
Screenshots were taken by building and running the microsite locally.

microsite

#### :heavy_check_mark: Checklist

 
- [x] Added documentation for Digital-ai Deploy plugin ([more info](https://docs.digital.ai/bundle/devops-deploy-version-v.24.1/page/deploy/concept/xl-deploy-backstage-overview.html))
- [x] Tests for new functionality.
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Comment by sishwarya** [comment]
Hi @Rugvip, and @camilaibs,  Could you please review the PR?

File: microsite/data/plugins/deploy.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+title: Deploy
```
Comment (freben) [inline microsite/data/plugins/deploy.yaml:2]: Name is too generic - could be `Digital.ai Deploy` for example? Same with the naming of this file and the image below.

**Review by freben — COMMENTED** [review]
Hi!

File: microsite/data/plugins/deploy.yaml (hunk 1 above)
Comment (sishwarya) [reply microsite/data/plugins/deploy.yaml]: Hi @freben , 
Thanks for reviewing the PR.
We have updated the name.

**Review by freben — APPROVED** [review]
Thanks!

**Comment by nelson-digi** [comment]
This repository is currently public.

**Pull request merged**